openxr/src/environment_depth.rs

```diff
@@ -0,0 +1,339 @@
+//! The [Environment Depth feature](https://developer.oculus.com/documentation/native/android/mobile-depth/)
+//! allows you to access depth maps based on the user environment.
+//! This is primarily used to occlude virtual objects by real-world objects and surfaces.
+//!
+//! This feature is exclusive to the Oculus Quest 3 as of September 2024.
+//!
+//! More details about the environment depth feature can be found in the [OpenXR specification](https://registry.khronos.org/OpenXR/specs/1.1/html/xrspec.html#XR_META_environment_depth).
+use crate::{
+    cvt, raw, sys, Graphics, Result, SystemId, Instance, Session, SessionInner, Space, View, Time
+};
+use std::marker::PhantomData;
+use std::ptr;
+use std::sync::Arc;
+
+#[derive(Debug, Copy, Clone)]
+pub struct SystemEnvironmentDepthProperties {
+    pub supports_environment_depth: bool,
+    pub supports_hand_removal: bool,
+}
+
+impl Instance {
+    #[inline]
+    pub fn supports_environment_depth(&self, system: SystemId) -> Result<SystemEnvironmentDepthProperties> {
```

Comment:
This doesn't return a bool, so "supports" might be an odd name choice. Maybe `environment_depth_properties`?

Reply:
Yes, that makes more sense.